app/routers/event.py

```diff
@@ -25,16 +27,66 @@ async def eventedit(request: Request):
                                       {"request": request})
 
 
-@router.get("/view/{id}")
-async def eventview(request: Request, id: int):
+@router.get("/view/{event_id}")
```

Comment:
Is `/view/` needed?

Reply:
Probably not particularly. I just like the uniformity, but I guess it could be removed

Reply:
@yammesicka Probably not particularly. I just like the uniformity, but I guess it could be removed